pkgs/tools/networking/litmus/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation rec {
+  version = "0.13";
+  name = "litmus-${version}";
+
+  src = fetchurl {
+    url = "http://webdav.org/neon/litmus/${name}.tar.gz";
+    sha256 = "09d615958121706444db67e09c40df5f753ccf1fa14846fdeb439298aa9ac3ff";
+  };
+
+  buildInputs = [ openssl ];
+  configureFlags = [ "--with-ssl" ];
+  installFlags = [ "PREFIX=$(out)" ];
```

Comment:
```suggestion
  installFlags = [ "PREFIX=${placeholder "out"}" ];
```

Reply:
Why?

Reply:
in this case it doesn't matter, but it's to ensure that you will get the storepath